nexodus-io/nexodus
# Implements automatically setting up nexd proxy ingress/egress rules using skupper config.

To test out the PoC:
1. get local dev server running using `make run-on-kind`.
2. run `./hack/skupper/install-skupper.sh` to install skupper and the nexd routers.  They will be installed to both a `east` and `west` namespace.
3. run `./hack/skupper/proxy-status.sh` to see the ingress/egress rules that get created on each nexd running in proxy mode.
